src/backend/mac/menu.rs

```diff
@@ -159,7 +159,7 @@ impl HotKey {
             KbKey::F19 => "\u{F716}",
             KbKey::F20 => "\u{F717}",
             _ => {
-                eprintln!("no key equivalent for {:?}", self);
+                eprintln!("no key equivalent for {self:?}");
```

Comment:
That requires Rust 1.58 but I couldn't find any mention of the MSRV, so I guess this should be fine?
If that's not the case, the `uninlined_format_args` clippy lint should be disabled instead.

Reply:
It's not explicitly defined, but right now during heavy development we're basically fine with treating whatever is currently the stable version as our MSRV.